web/styles/portico/markdown.css

```diff
@@ -368,14 +372,13 @@
         unicode-bidi: embed;
         direction: ltr;
         color: hsl(0, 0%, 0%);
-        border: 1px solid hsl(240, 13%, 90%);
         border-radius: 3px;
         /* Different from rendered_markdown.css */
```

Comment:
Is this comment meant to be deleted or adjusted here?

Reply:
It needs to be adjusted and made more specific/clear. These rules are different from the base rules for code elements in `rendered_markdown.css` (without the `rendered_markdown` class).